### PR TITLE
WIP:Make home page

### LINKED ISSUE #6: Make home page

### PR DESCRIPTION
**Pull request by biggicode**

### CODE REVIEW

File: src/component/auctions/auction.component.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export const Auction = () => {
+  return(
+    <div>
+      
+    </div>
```
Comment (QuintyHH) [inline src/component/auctions/auction.component.jsx:7]: Whats this?

Comment (biggicode) [reply src/component/auctions/auction.component.jsx:7]: This will be a page which will contain all auctions

**Review by Nicolleta1981 — APPROVED** [review]
:)

File: src/component/home/home.style.js [hunk 2]
```diff
@@ -66,3 +66,45 @@ export const ViewAllButton = styled.button`
...
+export const HowItWorksSection = styled.div`
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 100px;
+  padding: 50px;
+`
+
+export const HowItWorksCard = styled.div`
+
+`
```
Comment (QuintyHH) [inline src/component/home/home.style.js:85]: Too many properties.

**Review by QuintyHH — APPROVED** [review]
Overall good, with 2 comments

**Review by marianzburlea — COMMENTED** [review]
:) super

**Pull request merged**